samples/HealthChecksSample/LivenessProbeStartup.cs

```diff
@@ -53,11 +52,8 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             // The liveness check uses an 'identity' health check that always returns healty
             app.UseHealthChecks("/health/live", new HealthCheckOptions()
             {
-                // Filters the set of health checks run by this middleware
-                HealthCheckNames =
-                {
-                    "identity",
-                },
+                // Exclude all checks, just return a 200.
+                Predicate = (check) => false,
```

Comment:
Changing this to an arbitrary predicate rather than an explicit 'filter by name' seems like a good usability improvement. Since the set of checks has to be resolved per-request a lot of motivations for providing 'filter by name' seemed to melt away.